foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/DocumentTranslator.java

```diff
@@ -16,19 +21,28 @@
  * Created by santanu.s on 24/11/15.
  */
 public class DocumentTranslator {
-    private static final String CURRENT_RAW_KEY_VERSION = "__RAW_KEY_VERSION_2__";
 
     private static final Logger logger = LoggerFactory.getLogger(DocumentTranslator.class);
 
+    private String rawKeyVersion;
```

Comment:
Yeah but this is set through dropwizard configuration file which requires empty constructor.